PayKids/presentation/src/main/java/com/paykids/presentation/view/quiz/QuizShortAnswerImgFragment.kt

```
@@ -193,4 +193,28 @@ class QuizShortAnswerImgFragment : BaseFragment<FragmentQuizShortAnswerImgBindin
             }
         }
     }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        backPressedCallback = object : OnBackPressedCallback(true) {
+            override fun handleOnBackPressed() {
+            }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)